doc/developer/design/20220413_cluster_replica.md

```diff
@@ -0,0 +1,109 @@
+# Cluster replicas
+
+## Summary
+
+This document proposes an explicit SQL API for managing
+[cluster replicas](../platform/ux.md#cluster-replica).
+
+## Design
+
+### SQL syntax and semantics
+
+The `ALTER CLUSTER` statement will be removed.
+
+The `CREATE CLUSTER` and `CREATE CLUSTER REPLICA` statement will be evolved as
+follows:
+
+```sql
+create_cluster_stmt ::=
+  CREATE CLUSTER [IF NOT EXISTS] <name>
+    [<inline_replica> [, <inline_replica> ...]]
+
+create_cluster_replica_stmt ::=
+  CREATE CLUSTER REPLICA [IF NOT EXISTS] <name> OF <cluster>
```

Comment:
`OF` or `FOR`? The UX document uses `FOR`.

Reply:
Oops! I settled on `FOR`, but missed this one. Fixed.